src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -144,5 +160,55 @@ private bool TryReadFromSettings(string settingsKey, string defaultValue, out st
             settingsValue = _nameResolver.Resolve(settingsKey);
             return !string.IsNullOrEmpty(settingsValue);
         }
+
+        private static async Task CreateIfNotExistAsync(IDocumentDBService service, string databaseName, string collectionName, int throughput)
+        {
+            await DocumentDBUtility.RetryAsync(() => CreateDatabaseIfNotExistsAsync(service, databaseName),
+                DefaultMaxThrottleRetries, codesToIgnore: HttpStatusCode.Conflict);
```

Comment:
Do you know if these operations are retried internally by the DocumentClient? When I added this `RetryAsync` utility, the client didn't do any retries. But now this may not be necessary. I'd like to eventually remove it if we know that retries are handled internally.